component-maps.yml

```diff
@@ -501,6 +519,15 @@ docker_image:
 
 docker_container:
 
+  mender-gateway:
+    git:
+      - integration
```

Comment:
`integration`? It's not built there, is it?

Reply:
There is no `mender-gateway` Docker container at all, so this whole block can be removed.